views/render-identify.php

```diff
@@ -0,0 +1,8 @@
+<script type="text/javascript">
+    metrilo.identify("<?php echo $this->identify_call_data; ?>");
+    <?php
+        if ($this->user_tags) {
+            echo "window.metrilo.applyTags($this->user_tags)";
+        }
+    ?>
```

Comment:
These should be wrapped in guarding script that checks if our library is loaded first (if we finally make the library non-blocking for the client's browser).
```javascript
  (function() {
    const f = function () {
        metrilo.identify("<?php echo $this->identify_call_data; ?>");
        <?php
          if ($this->user_tags) {
              echo "window.metrilo.applyTags($this->user_tags);";
          }
        ?>
    };
    if (typeof window.metrilo !== 'undefined') {
      f();
    } else {
      window.metriloQueue = window.metriloQueue || [];
      window.metriloQueue.push(f);
    }
  })();
```